index.bs

```diff
@@ -3093,6 +3096,43 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+## Biometric Authenticator Matching Criteria Extension (biometricBound) ## {#sctn-authenticator-biometric-criteria-extension}
+
+This [=registration extension=] allows a [=[RP]=] to specify the desired performance of a biometric authenticator
+that could be leveraged when creating the credential. 
+
+: Extension identifier
+:: `biometricMatchingCriteria`
+
+: Client extension input
+:: Biometric performance bounds:
+
+    <pre class="idl">
+    dictionary authenticatorBiometricMatchingCriteria{
+        float FAR;
+        float FRR;
+        };
+    </pre>
```

Comment:
IIUC, we ought to define the input in terms of CDDL, rather than WebIDL, yes?
see also issue #679 "add CDDL to every extension" and issue #626 "Extensions need to define how their parameters convert to/from CBOR"